specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/stable/2019-09-16/healthcare-apis.json

```diff
@@ -681,6 +681,24 @@
         "etag": {
           "description": "An etag associated with the resource, used for optimistic concurrency when editing it.",
           "type": "string"
+        },
+        "identity": {
```

Comment:
adding new property to existing definition will be breaking change in a stable version,  suggest to add  in a new version.

Reply:
We are open to creating a new version, but were also hoping to support managed identity for users who have deployed the current version of our service.  It is possible to add this property without updating the version?

Reply:
"identity" is a special case as it's an ARM-managed property (outside of the custom RP "properties"). In this case I think we can make an exception and allow it - I'll sign off for ARM.

Reply:
thanks @anthony-c-martin for clarification. it's safe to merge then.